odata-client-runtime/pom.xml

```diff
@@ -90,4 +91,25 @@
 
     </dependencies>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <configuration>
+                    <obrRepository>NONE</obrRepository>
+                    <instructions>
+                        <Export-Package>
+                            com.github.davidmoten.odata.client,
+                            com.github.davidmoten.odata.client.annotation,
+                            com.github.davidmoten.odata.client.edm,
+                            com.github.davidmoten.odata.client.internal, <!-- Guess internal doesn't really mean internal -->
```

Comment:
internal packages should not be exported. The classes are still available at runtime but not for direct use.

Reply:
why not use * includes and exclude internal?

Reply:
> internal packages should not be exported. The classes are still available at runtime but not for direct use.

Because of this warning: `Export com.github.davidmoten.odata.client,  has 1,  private references [com.github.davidmoten.odata.client.internal]`

There are public references to `com.github.davidmoten.odata.client.internal.TypedObject` in at least one class (e.g. the constructor for ActionFunctionRequestBase)

Until all public references are removed, the internal packages need to be exported or the module will fail to resolve when loaded into an OSGI runtime.

> why not use * includes and exclude internal?

That can be done, although it would be better if the exports weren't specified at all and bndtools was left to work it out itself.  If we can sort out the public references to internal packages then the whole plugin config entry can be removed.

Reply:
thanks for the explanation. You can include the internal package for the moment and I've opened #8 to deal with it later.